accelerator-core/src/main/java/com/tokbox/android/otsdkwrapper/wrapper/OTWrapper.java

```diff
@@ -1159,13 +1166,12 @@ public void onError(Session session, OpentokError opentokError) {
     public void onConnectionCreated(Session session, Connection connection) {
       LOG.d(LOG_TAG, "onConnectionCreated: ", connection.getData());
       mSession.connections.put(connection.getConnectionId(), connection);
-      mConnectionsCount++;
       if (connection.getCreationTime().compareTo(mSessionConnection.getCreationTime()) <= 0) {
         mOlderThanMe++;
       }
       if (mBasicListeners != null) {
         for (BasicListener listener : mBasicListeners) {
-          ((RetriableBasicListener) listener).onConnected(SELF, mConnectionsCount,
+          ((RetriableBasicListener) listener).onConnected(SELF, mSubscribers.size(),
```

Comment:
I don't think we should replace the number of connections by number of subscribers